C/plugins/storage/sqlitelb/common/readings.cpp

```diff
@@ -1635,6 +1635,8 @@ unsigned int  Connection::purgeReadings(unsigned long age,
 	result += " \"unsentPurged\" : 0, ";
 	result += " \"unsentRetained\" : 0, ";
 	result += " \"readings\" : 0 }";
+	result += " \"method\" : \"time\", ";
+	result += " \"duration\" : 0 }";
```

Comment:
Invalid JSON structure.
Fixed in commit https://github.com/fledge-iot/fledge/pull/1166/commits/efe86c9cb3979f8d2012ca2cf0cd6c03ec3ee107